### DISCUSSION TITLE
Masking Sensitive data like passwords when commenting to GitHub

### DISCUSSION THREAD

**Discussion by vishnu-techprescient-teika** [suzuki-shunsuke/tfcmt#1082] [Deprecated (Feature Request)]
### Feature Overview

Masking Sensitive data like passwords when commenting to GitHub

### Why is the feature needed?

Terraform masks sensitive data automatically but there are cases where it is still unable to do it. 
Like :
- As a part of the database connection URL.
- As a part of some yaml script.

Sometimes these resources as coded in such a fashion Terraform misses it or we want to check part of the resource but will only get `sensitive` in place. 
If we can mask the same before pushing it to Github that would be great.

### Does the feature include Breaking Changes?

No, it should not have breaking changes. Tfcmt is parsing terraform output before posting to Github, we should be able to add a regex match and replace function easily, with a switch.

### Example Code

```console
$ 
```

Configuration

```yaml

```

### Reference

*

[a comment by an automated account is not shown]

**Comment by suzuki-shunsuke** [comment]
Thank you for creating a discussion. https://github.com/suzuki-shunsuke/tfcmt/discussions/1083
I close this discussion.
